src/CommitmentsV2/SFA.DAS.CommitmentsV2/DependencyResolution/ValidatorRegistry.cs

```diff
@@ -0,0 +1,17 @@
+﻿using MediatR;
+using StructureMap;
+
+namespace SFA.DAS.CommitmentsV2.DependencyResolution
+{
+    public class ValidatorRegistry : Registry
+    {
+        private const string ServiceName = "SFA.DAS.CommitmentsV2";
+
+        public ValidatorRegistry()
+        {
+            // The validators are registered automatically by a call to AddFluentValidation in startup
+
+            For(typeof(IPipelineBehavior<,>)).Use(typeof(ValidationBehavior<,>));
```

Comment:
This Validator is tightly bound to the Mediator, should it not be in the MediatorRegistry